dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/insertmanager/insertionfilters/Base64Filter.java

```diff
@@ -0,0 +1,53 @@
+package org.dpppt.backend.sdk.ws.insertmanager.insertionfilters;
+
+import java.util.List;
+import org.dpppt.backend.sdk.model.gaen.GaenKey;
+import org.dpppt.backend.sdk.semver.Version;
+import org.dpppt.backend.sdk.utils.UTCInstant;
+import org.dpppt.backend.sdk.ws.insertmanager.InsertException;
+import org.dpppt.backend.sdk.ws.insertmanager.OSType;
+import org.dpppt.backend.sdk.ws.util.ValidationUtils;
+
+/**
+ * All keys must be valid Base64 encoded. Non valid Base64 keys are not allowed and are filtered
+ * out. This filter rejects the whole submitted batch of keys, if any of the keys is not valid
+ * Base64, as this is a client error.
```

Comment:
Nicely done on documenting this! I was at first wondering about the different way this filter worked.